robotouille/robotouille_simulator.py

```diff
@@ -10,18 +10,18 @@ def simulator(environment_name: str, seed: int=42, noisy_randomization: bool=Fal
     renderer.render(obs, mode='human')
     done = False
     interactive = False # Set to True to interact with the environment through terminal REPL (ignores input)
-
+    
     while not done:
         # Construct action from input
         pygame_events = pygame.event.get()
         # Mouse clicks for movement and pick/place stack/unstack
         mousedown_events = list(filter(lambda e: e.type == pygame.MOUSEBUTTONDOWN, pygame_events))
         # Keyboard events ('e' button) for cut/cook ('space' button) for noop
         keydown_events = list(filter(lambda e: e.type == pygame.KEYDOWN, pygame_events))
-        action, args = create_action_from_control(env, obs, mousedown_events+keydown_events, renderer)
+        action, args = create_action_from_control(env, obs, obs.current_player, mousedown_events+keydown_events, renderer)
         if not interactive and action is None:
             # Retry for keyboard input
             continue
-        obs, reward, done, info = env.step(action=action, args=args, interactive=interactive)
+        obs, reward, done, info = env.step(actions={action: args}, interactive=interactive)
```

Comment:
We would want the `actions` to be a list whose length is the number of players for this environment and whose elements are NOOP if you are not controlling the character or the current player's action + args if it is. This will make it easier to implement realtime.

Reply:
Updated actions